simulation_parameters/microbe_stage/bio_processes.json

```diff
@@ -214,11 +214,11 @@
     "Name": "PHOTOSYNTHESIS",
     "Inputs": {
       "sunlight": 1,
-      "carbondioxide": 1
+      "carbondioxide": 0.081
     },
     "Outputs": {
-      "glucose": 0.012,
-      "oxygen": 1
+      "glucose": 0.036,
+      "oxygen": 0.081
```

Comment:
I believe that glucose production has to be increased x4 here, as otherwise it matches thylakoids' performance

Reply:
I multiplied both chloroplast process and thylakoid process output by 3x, so the ratio should be the same. The first photosynthesis process has output of: `0.12` and the thylakoid has: `0.036`. So it is still multiple times more, so isn't it fine or what numbers do you suggest using?

Reply:
Glucose production depends on the ratio between carbondioxide and glucose values. So, normal photosynthesis reaches 100% performance at 27% CO2 and 0.12 glucose production.
Chromatophore photosynthesis reaches 333.(3)% performance at the same level of CO2 and 3.(3) * 0.036 = 0.12 glucose, which is the same as normal photosynthesis.